Umbraco-Heartcore/Client-Libraries/Dot-Net-Console/index.md

```diff
@@ -0,0 +1,103 @@
+---
+versionFrom: 8.0.0
+---
+
+# .NET Core Console Application
+
+In this article, you can read more about the .NET Core Console Application.
+
+We will go through the process of setting the application up, explore what you can do with the application and how you can connect to your Heartcore project on Umbraco Cloud.
+
+:::note
+In order to use this console application, you will need to have the .NET Core SDK2.2. Older or newer versions will __not work__ with the application.
+:::
+
+## Installing the Console Application
+
+Once you have cloned down the [Umbraco Headless Client](https://github.com/umbraco/Umbraco.Headless.Client.Net), there are two ways of running the application.
```

Comment:
These "two ways of running the application", those are the next in this article?
So:
1: Using Visual Studio and
2: Using the command line

Is that correctly understood?
If it is, perhaps you could add the numbers to the headings as well? 🤔 Just an idea!